src/node/history.h

```diff
@@ -526,21 +526,26 @@ namespace ccf
       {
         auto version = store.next_version();
         auto view = consensus->get_view();
-        auto commit = consensus->get_commit_seqno();
+        auto commit_txid = consensus->get_committed_txid();
         LOG_DEBUG_FMT("Issuing signature at {}", version);
         LOG_DEBUG_FMT(
-          "Signed at {} view: {} commit: {}", version, view, commit);
+          "Signed at {} in view: {} commit was: {}.{}",
```

Comment:
Note that `view` != `commit_view`, across view changes. `view` is the view as of the signature, and is useful to the view history replication across nodes. `commit_view` is the view of the last commit at the time of that signature (`commit_seqno`).